tests/countries/test_bonaire_sint_eustatius_and_saba.py

```
@@ -0,0 +1,236 @@
+            "2023-05-28",
+            "2024-05-19",
+            "2025-06-08",
+        )
+        self.assertHolidayName(name, dt)
+
+    def test_emancipation_day(self):
+        name = "Dag van de Emancipatie"
+        self.assertNoHolidayName(name)
+
+        for subdiv, holidays in self.subdiv_holidays.items():
+            if subdiv == "BON" or subdiv == "STA":
```

Pylint (3.3.7)

[refactor] Consider merging these comparisons with 'in' by using 'subdiv in ('BON', 'STA')'. Use a set instead if elements are hashable.

(R1714)